dev.yml

```diff
@@ -3,8 +3,7 @@ name: enterprise-script-service
 up:
   - xcode_clt
```

Comment:
Do we need to do something about this?

Some folks were getting errors about this being an invalid task.

Reply:
Also tried running this branch with this line removed and I'm getting some odd output from `dev up` (it succeeds but there's a message saying `bundle install Failed!` as part of its output) and `dev compile` fails with `ArgumentError: Trying to register Bundler::GemfileError for status code 4 but Bundler::GemfileError is already registered`

Reply:
Got rid of that error by removing the top line in `Rakefile`. I could then `dev up`, `dev compile`, and copying the executable and scripts were executing in core.

Reply:
I removed the Xcode line and I don't seem to have an issue with `dev up` and `dev compile` 🤔  Can you retry with the latest version of this branch?

Reply:
I'm still getting an error with `dev compile`:

```
➜  ess git:(bump-development-gems-versions) dev compile
👩‍💻  Running bin/rake compile from dev.yml
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:105: warning: already initialized constant Bundler::FileUtils::VERSION
/opt/rubies/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:105: warning: previous definition of VERSION was here
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1284: warning: already initialized constant Bundler::FileUtils::Entry_::S_IF_DOOR
/opt/rubies/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1284: warning: previous definition of S_IF_DOOR was here
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1568: warning: already initialized constant Bundler::FileUtils::Entry_::DIRECTORY_TERM
/opt/rubies/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1568: warning: previous definition of DIRECTORY_TERM was here
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1626: warning: already initialized constant Bundler::FileUtils::OPT_TABLE
/opt/rubies/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1626: warning: previous definition of OPT_TABLE was here
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1685: warning: already initialized constant Bundler::FileUtils::LOW_METHODS
/opt/rubies/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1685: warning: previous definition of LOW_METHODS was here
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/vendor/fileutils/lib/fileutils.rb:1692: warning: already initialized constant Bundler::FileUtils::METHODS
/opt/rubies/2.7.1/lib/ruby/2.7.0/bundler/vendor/fileutils/lib/fileutils.rb:1692: warning: previous definition of METHODS was here
rake aborted!
ArgumentError: Trying to register Bundler::GemfileError for status code 4 but Bundler::GemfileError is already registered
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/errors.rb:9:in `status_code'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/errors.rb:20:in `<class:GemfileError>'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/errors.rb:20:in `<module:Bundler>'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/errors.rb:3:in `<top (required)>'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler.rb:7:in `require_relative'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler.rb:7:in `<top (required)>'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/gem_helper.rb:3:in `require_relative'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/gem_helper.rb:3:in `<top (required)>'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/gem_tasks.rb:6:in `require_relative'
/Users/jeffreycharles/.gem/ruby/2.7.1/gems/bundler-2.1.4/lib/bundler/gem_tasks.rb:6:in `<top (required)>'
/Users/jeffreycharles/src/github.com/Shopify/ess/Rakefile:1:in `require'
/Users/jeffreycharles/src/github.com/Shopify/ess/Rakefile:1:in `<top (required)>'
(See full trace by running task with --trace)
```

Reply:
I can't seem to reproduce this error. It could be because my machine is in a clean state (I only cloned this repo last week). I did notice that `dev up` didn't `bundle install` for me, so I manually ran that before running `dev compile`. Maybe some of your packages are out of date and a manual `bundle install` is required?

Reply:
Yes, I think it's due to a stale Gemfile.lock. The lockfile is not comitted in ESS and sometime gets stale (especially here considering we changed the ruby version too).

Reply:
Spoke offline with Jeff. Looks like this is an environmental issue. If the bundler path is override (e.g. to `vendor/bundle`), then you don't get this error. I've opened [this ticket](https://github.com/Shopify/script-editor/issues/2320) to investigate fixing this since it's outside the scope of this PR.

We should be good to 🚢  this!